opencensus/stats/exporters/stackdriver_exporter.py

```diff
@@ -233,23 +231,25 @@ def create_time_series_list(self, v_data, option_resource_type,
                 # [0, first_bound).
                 bounds.extend([0])
                 buckets.extend([0])
-                bounds.extend(list(map(float, agg_data.bounds)))
-                buckets.extend(list(map(int, agg_data.counts_per_bucket)))
-            elif isinstance(agg,
-                            aggregation.aggregation_data.CountAggregationData):
+                bounds.extend(list(map(float, agg.bounds)))
+                buckets.extend(list(map(int, agg.counts_per_bucket)))
+            elif aggregation_type is aggregation.Type.COUNT:
                 point.value.int64_value = agg.count_data
-            elif isinstance(
-                    agg, aggregation.aggregation_data.SumAggregationDataFloat):
-                point.value.double_value = agg.sum_data
-            elif not isinstance(
-                    agg, aggregation.aggregation_data
-                    .LastValueAggregationData):  # pragma: NO COVER
+            elif aggregation_type is aggregation.Type.SUM:
+                if isinstance(v_data.view.measure, measure.MeasureInt):
+                    # TODO: Add implementation of sum aggregation that does not
+                    # store it's data as a float.
+                    point.value.int64_value = int(agg.sum_data)
+                if isinstance(v_data.view.measure, measure.MeasureFloat):
+                    point.value.double_value = float(agg.sum_data)
+            elif aggregation_type is aggregation.Type.LASTVALUE:
                 if isinstance(v_data.view.measure, measure.MeasureInt):
                     point.value.int64_value = int(agg.value)
-                elif isinstance(v_data.view.measure, measure.MeasureFloat):
+                if isinstance(v_data.view.measure, measure.MeasureFloat):
                     point.value.double_value = float(agg.value)
             else:
-                point.value.string_value = str(tag_value[0])
+                raise Exception("unsupported aggregation type: %s" %
```

Comment:
Is it okay to throw an `Exception` at this point in the export? Seems it would be an artifact of their view being improperly configured, and should've already been thrown in `create_metric_descriptor`. Even if we shouldn't thrown an exception, it didn't make sense to me to set the value to that of the first tag.